### DISCUSSION TITLE
can someone tell me what the terms discovery, delivering and in review mean?

### DISCUSSION THREAD

**Discussion by Stefandro90** [Leantime/leantime#535] [Q&A]
?

**Comment by marcelfolaron** [comment]
We have a documentation on that topic in https://docs.leantime.io/#/knowledge-base/idea-development

Basically we are evaluating features in terms of their stage:

Discovery: You are currently performing research on the problem and gathering data. This step would be required to start delivering
Delivering: You are now working on the feature.
In Review: The feature is complete and someone else is reviewing it (client, manager etc)

**Reply by Stefandro90** [reply]
Thank you for your help :)
